modules/EmailTemplates/EditView.php

```diff
@@ -113,7 +113,7 @@
 if (!$focus->ACLAccess('EditView')) {
     ACLController::displayNoAccess(true);
     sugar_cleanup(true);
-} elseif (!is_admin($current_user) && $focus->type === 'system') {
+} elseif (!is_admin($current_user) && isset($focus->type) && $focus->type === 'system') {
```

Comment:
Works for me but it should be combined with the previous if statement as the code in the body is duplicated.